### PR TITLE
Fixes #7520: package_install* methods do not work outside of debian|redhat|SuSE Linux distributions and don't warn about it

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7520

### CODE REVIEW

File: tree/30_generic_methods/package_install_version_cmp_update.cf [hunk 1]
```diff
@@ -171,4 +171,9 @@ bundle agent package_install_version_cmp_update(package_name, version_comparator
       "Reports" usebundle => _logger("Verify that package ${package_name} is installed and up-to-date", "${class_prefix}");
     !immediate.version_not_specified.!update::
       "Reports" usebundle => _logger("Verify that package ${package_name} is installed", "${class_prefix}");
+
+    !debian.!redhat.!SuSE::
```
Comment (peckpeck) [inline tree/30_generic_methods/package_install_version_cmp_update.cf]: shouldn't we add (debian|redhat|suse) in previous conditions ?

**Comment by jooooooon** [comment]
PR updated

File: tree/30_generic_methods/package_install_version_cmp_update.cf (hunk 1 above)
Comment (jooooooon) [inline tree/30_generic_methods/package_install_version_cmp_update.cf]: Yes, absolutely! PR updated.

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**